Models/IVolunteerUser.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Karma.Models
+{
+    public interface IVolunteerUser
+    {
+        Guid Id { get; set; }
+        string Name { get; set; }
+        string Surname { get; set; }
+        List<CharityEvent> ParticipatedEvents { get; set; }
```

Comment:
This should be a list of guids (those guids would be the charity event guids).
It will be a lot more universal (if for example you want to have fundraiser events that have volunteers in the future)

Reply:
The list now contains only charity event Id's